fix some defaults

grant2258 opened the pull request:
Im changing a few defaults ill give a reason for each one.

Artwork resolution multiplier  changes to 1x scaling by default.

number one there is no benefit to setting this to 2x and is slows older hardware down. Vector games automatically set this to 1 is if its set to 2.  there is actually 3 option auto 0, 1,2 


Vector resolution multiplier changed to 3
This is the setting people usually have to end up setting to get fine looking lines with bi-linear filtering on or an appropriate shader. Lower res done usually work too well.

Bypass audio skew (Restart core) changed back to what its always been no idea why this changed.

I dont have not problem defaulting this to on when RA comes with a default audio skew that plays games at the right and audio speed when its not 60fps. Bypassing this is playing all games at the right speed and works without any user intervention and should be left until/if RA changes it to a default that works without users having to search on how to fix it . Its a feature not have use thing and playing games at the wrong speed by default isint good.

markwkidd commented:
> number one there is no benefit to setting this to 2x and is slows older hardware down.

Having the artwork multiplier to 2x definitely makes an improvement with `gorf`. Does having 2x by default cause a problem with any games?

grant2258 commented:
yea it does on slower systems mark that start loosing fps you can change it back if you like mark its not something im insisting on. None of the setting are just giving reasons why i changed them so you know where im hitting it from.

markwkidd commented:
OK, that makes sense. I'm working by phone now so if you can allow me a bit
of time to look at this and think about the universe before I merge.

On Fri, Jan 18, 2019 at 9:57 PM grant2258 <notifications@github.com> wrote:

> yea it does on slower systems mark that start loosing fps you can change
> it back if you like mark its not something im insisting on. None of the
> setting are just giving reasons why i changed them so you know where im
> hitting it from.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/libretro/mame2003-plus-libretro/pull/598#issuecomment-455743463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASphdkvbYdPB8Dix_uUS56LlpehRS6v5ks5vEomWgaJpZM4aI544>
> .
>


-- 
Mark W. Kidd
http://facebook.com/markwkidd
(606)536-0115

grant2258 commented:
Ok i changed this pr to only fix the machine timing to whats its always been. The other two setting can go in another pr this one needs set back to its defualt since the day i fixed this. 

The only reason I added this was for aussiebloke to test mac problems (i was something else that was the cause). You must have changed it by accident when syncing both mames up.
